import_craft.py

```diff
@@ -114,13 +114,14 @@ def smooth_object_meshes(self, ob):
     def read_parts_models(self, prefabs_dict, colliders, allow_no_material_mesh):
         for part in prefabs_dict.values():
             unselect_all_objects()
+
             result = import_mu.import_mu(self, bpy.context, part['mu'], False)
             if not result == {'FINISHED'}:
-                print('Warning : Error while importing file {}'.format(part['mu']))
-                return
-            part['object'] = bpy.context.scene.objects.active
-            self.smooth_object_meshes(part['object'])
-            self.rename_data_elements(part['object'])
+                print('Warning: Error while importing file {}'.format(part['mu']))
```

Comment:
you are no longer returning. Are you sure all subsequent method will work in this case?